packages/grafana-ui/src/components/BigValue/BigValueLayout.tsx

```diff
@@ -111,8 +116,8 @@ export abstract class BigValueLayout {
   ): PercentChangeStyles {
     const VALUE_TO_PERCENT_CHANGE_RATIO = 2.5;
     const valueContainerStyles = this.getValueAndTitleContainerStyles();
-    const percentFontSize = Math.max(this.valueFontSize / VALUE_TO_PERCENT_CHANGE_RATIO, 12);
-    let iconSize = Math.max(this.valueFontSize / 3, 10);
+    const percentFontSize = this.percentFontSize || Math.max(this.valueFontSize / VALUE_TO_PERCENT_CHANGE_RATIO, 12);
+    let iconSize = this.percentFontSize ? this.percentFontSize - 3 : Math.max(this.valueFontSize / 3, 10);
```

Comment:
The `this.percentFontSize - 3` seems quite arbitrary, will it work with any font size?

Reply:
Yes, I did that to ensure the size of the icon is consistent with the regular auto-calculated icon size (the behavior you got before I made any changes, or when leaving the font size empty or 0)